static/js/settings.js

```diff
@@ -821,6 +821,12 @@ exports.update_page = function () {
     i18n.ensure_i18n(_update_page);
 };
 
+exports.update_email = function () {
+    var message = i18n.t('We have sent your confirmation email.');
+    viewport.scrollTop(0);
```

Comment:
I have changed this. I copied `viewport.scrollTop(0);` from `settings_change_success` function in 'static/js/settings.js'.